test/client/client.go

```diff
@@ -253,6 +254,9 @@ func (c *Client) Host() string { return c.cfg.Host }
 // ControllerRuntimeClient returns the underlying controller runtime Client
 func (c *Client) ControllerRuntimeClient() crclient.Client { return c.c }
 
+// HTTPClient returns a plain http.Client with same config as this client.
+func (c *Client) HTTPClient() (*http.Client, error) { return rest.HTTPClientFor(c.cfg) }
```

Comment:
I assume we no longer need this?

Reply:
You're right - I can construct that client in the e2e test without messing with test.Client(). Fixed next push.